store/tikv/tikvrpc/tikvrpc.go

```diff
@@ -61,6 +62,20 @@ const (
 	CmdMvccGetByKey CmdType = 1024 + iota
 	CmdMvccGetByStartTs
 	CmdSplitRegion
+
+	CmdDebugGet CmdType = 2048 + iota
```

Comment:
Do we need this? I guess the order does not matter here.

Reply:
Yeah, but it's clear to keep order here, also no other impaction.

Reply:
`String` and `IsDebugReq` will be called for every request, it is better to reduce the useless work.

Reply:
All right.